tendrl/node_agent/message/logger.py

```diff
@@ -0,0 +1,81 @@
+import logging
+from tendrl.commons.message import Message
+
+
+LOG = logging.getLogger(__name__)
+
+
+class Logger(object):
+    logger_priorities = {"notice": "info",
+                         "info": "info",
```

Comment:
@nthomas-redhat alert are always comes as notice, anmol can pick the alert messages from etcd which is in notice type. 

We are storing all the messages in  etcd under /Messages. Anmol can tack alert from etcd using which have type notice.  But in log there is no notice function so i logging notice as info